CoEpi/repo/CENKeyRepo.swift

```diff
@@ -0,0 +1,11 @@
+protocol CENKeyRepo {
```

Comment:
This should all be rx, but doesn't have to be in this PR

Reply:
How would we make this reactive?

Reply:
For "setters" we would return `Completable` and for "getters" ideally we'd maintain observable fields in the repo, such that potentially multiple observers (e.g. different screens) can subscribe to them.

Reply:
On the other side, keeping the api synchronous isn't necessarily bad, as it can be also easily composed with other synchronous apis. The threading / rx would be added "outside".

Reply:
I see... lets do it outside of this PR then